source/fundamentals/indexes.txt

```diff
@@ -244,13 +306,9 @@ language as an option when creating the index.
 
 .. tip::
 
-   Text indexes differ from the more powerful
-   :atlas:`Atlas full text search indexes </atlas-search/?jmp=docs_driver_java>`.
-   Atlas users should use Atlas search.
-
-.. common-content-end
-
-.. driver-content-begin
+   Text indexes differ from the more fully-featured MongoDB Atlas Search
```

Comment:
S: change the wording of "more fully-featured" to be less redundant.

Reply:
I'll replace it with the text from https://www.mongodb.com/docs/manual/core/indexes/index-types/index-text/